models/source/im-bm-ci/im_ontap_flavor.sql

```diff
@@ -0,0 +1,15 @@
+{{
+  config({
+    "materialized" : "table",
+    "post-hook" : [
+      "grant select on table {{this}} to group non_gl_read_only"
+      ]
+    })
+}}
+
+select
+    cast(udf_ontap_flavor_id as int) as ontap_flavor_id,
+    udf_flavor as flavor,
+    udf_conc_type as conc_type,
```

Comment:
Sage has a limit on column name length, but we don't. When possible, spell out the full word to avoid ambiguity. So this should be concentration_type instead of conc_type